BLSQ/iaso
# POLIO-892: update Polio Budget teams fixtures

The team structure of latest config uses two more teams and teams of teams

Related JIRA tickets :POLIO-892

## Self proofreading checklist


## Changes

- split teams fixture
- update team fixture to fit new config
- update [WIKI](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/polio_budget_workflow)

## How to test

⚠️  if you have already existing teams it can override them
Run `docker-compose run iaso manage loaddata plugins/polio/fixtures/teams.yaml`
Make sure you have a structure similar to this:

![Screenshot 2023-03-15 at 11 19 18](https://user-images.githubusercontent.com/12494624/225289624-84c2d229-e09e-4712-bcef-16dd6af5f90f.png)
